libcnb/src/runtime.rs

```diff
@@ -54,7 +54,6 @@ pub fn libcnb_runtime<B: Buildpack>(buildpack: &B) {
         .and_then(Path::file_name)
         .and_then(OsStr::to_str);
 
-    #[cfg(any(target_family = "unix"))]
```

Comment:
I think removing this to allow compilation on non-unix systems is fine. However, since we're matching the unix specific binary names, this would never work on i.e. Windows where `detect` would be `detect.exe` for example.

Since we don't support Windows buildpacks this isn't a huge deal, but maybe we can add a comment here to remind our future selves about this?

Reply:
Actually, I think don't even need a comment here. Adding Windows support would require us to go over everything anyway and the executable name matching will be probably the most obvious thing to fix. :)